packages/scandipwa/src/style/abstract/_button.scss

```diff
@@ -28,7 +28,6 @@
     padding-inline: var(--button-padding);
     text-decoration: var(--button-text-decoration);
     text-transform: uppercase;
-    transition-duration: .25s;
     transition-timing-function: ease-out;
     transition-property: background-color, color, border;
```

Comment:
Please remove other transition properties, since they are artifacts and may cause issues later if we leave them here.